plugins/vpp/aclplugin/vppdump/dump_vppcalls.go

```diff
@@ -125,9 +128,9 @@ func DumpMACIPACL(swIfIndices ifaceidx.SwIfIndex, log logging.Logger, vppChannel
 	ruleMACIPData := make(map[ACLIdentifier][]*acl.AccessLists_Acl_Rule)
 
 	// get all ACLs with MACIP ruleData
-	MACIPRuleACLs, err := DumpMacIPAcls(log, vppChannel, stopwatch)
-	if err != nil {
-		return nil, err
+	MACIPRuleACLs, _ := DumpMacIPAcls(log, vppChannel, stopwatch)
+	if len(MACIPRuleACLs) < 1 {
+		return ACLs, nil
```

Comment:
Same thing here, just do `return nil, nil`. This makes it clear instantly when looking at this line that slice is empty without going back to check if there is any append before this.